deploy.sh

```diff
@@ -10,7 +10,7 @@ VPC_STACK_TEMPLATE=./provisioning/cloudformation/templates/workshop-vpc.yml
 SUBNET_CIDR=10.10.21.0/24
 VPC_CIDR=10.10.21.0/22
 OWASP_ZAP_VERSION=2.5.0
-IMAGE_ID=ami-97785bed  # latest amzn linux
+IMAGE_ID=ami-00eb20669e0990cb4  # latest amzn linux
```

Comment:
for any account/region specific stuff like this, be sure it's called out in the readme for how to bring this guy up again.

Reply:
Added  to the `Create Workshop Environment` section in the README for this